package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "3.20.0",
+  "version": "3.21.0",
```

Comment:
tried to use `npm update` but it didn't work. Once this is merged, do we publish this to npm?

Reply:
We've been using the git commit to pull it in:

`yarn remove vets-json-schema`
`yarn add https://github.com/department-of-veterans-affairs/vets-json-schema.git\#<commit hash>`